download/README.md

```diff
@@ -3,6 +3,7 @@ sidebar: false
 layout: AboutPage
 title: Download openHAB
 currentVersion: 2.3.0
+currentMilestoneVersion: 2.4.0.M7
```

Comment:
That's gonna be a bit nasty to always keep that up to date. We had a discussion somewhere else that we would need to consolidate the current version and download links at some common place. Could we make Jenkins publish the latest milestone version somewhere, where it could be picked up?

Reply:
I know - since I had no good solution in mind I kept using the frontmatter, since it's relatively easy to modify (by editing the file directly on GitHub for example). The current version could also be retrieved by the Ruby script during the build - I wouldn't go the "full-dynamic" route and make the client do the request, it's too risky.

Reply:
Another good alternative could be to have the milestone Jenkins builds themselves commit the new version to this repository (via openhab-bot) - this would retrigger a deployment by Netlify.

Reply:
Yes, something like this is probably the best. We will have to define some file format and a way for Jenkins to update it.